content/enterprise/requirements/data-storage/vault.mdx

```diff
@@ -1,98 +1,107 @@
 ---
-page_title: Externally Managed Vault - Terraform Enterprise
+page_title: External Vault - Terraform Enterprise
 ---
 
-# Externally Managed Vault Requirements for Terraform Enterprise
+# External Vault Requirements for Terraform Enterprise
 
-Terraform Enterprise uses Vault to encrypt sensitive information such as variables and state versions.
+Terraform Enterprise automatically creates an internally-managed Vault server
+that stores its data in the PostgreSQL Database. We strongly recommend that
+organizations use this internally-managed Vault server. However, some
+organizations have specific requirements around data encryption and auditing.
+Those organizations can configure Terraform Enterprise to use an external Vault
+server rather than the internally-managed Vault server.
```

Comment:
```suggestion
server rather than the internally-managed Vault server.

We only recommend using external Vault when you have experience managing Vault in production. This approach requires that you assume full responsibility for the Vault server, including sealing, unsealing, replication, etc.
```